packages/components/src/custom-select-control/style.scss

```
@@ -41,18 +41,12 @@
 		display: none;
 	}
 
-	// Block UI appearance.
-	border: $border-width solid $gray-900;
-	background-color: $white;
-	border-radius: $radius-block-ui;
 	outline: none;
 	transition: none;
-
 	max-height: 400px;
-	min-width: 100%;
 	overflow: auto;
 	padding: 0;
-	position: absolute;
+	margin: 0 -$border-width; // Allow width value to include the Popover border width.
```

```suggestion
	margin: 0 (-$border-width); // Allow width value to include the Popover border width.
```

I believe the Sass compiler treats `-$border-width` like a subtraction operation. 

So in the original code `margin: 0 -$border-width;` results in `margin: -1px;` not `margin: 0 -1px`. Given the comment only mentions the width I assume this is unintended.

---

Great catch, thanks Aaron! Yes, the negative margin is only intended for the horizontal margin. It's to reduce the width of the menu by 2px to account for the border of the parent, so that the popover lines up perfectly with the toggle button. I'll see if I can squeeze in a bit of time today to tweak it.